### PR TITLE
Updating PWA article on Window's package generation

### PR DESCRIPTION
**Pull request by MariaSolOsso**
Updated the article with the steps corresponding to the newest version of PWABuilder (as well as the image).

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/microsoft-store.md [hunk 1]
```diff
@@ -95,18 +95,19 @@ Now that you have your app publishing information, generate a Windows app packag
...
     *   **Publisher ID**  
     *   **Publisher Display Name**  
         
-    :::image type="complex" source="./media/pwabuilder-publisher-info.msft.png" alt-text="Paste publisher information into PWABuilder" lightbox="./media/pwabuilder-publisher-info.msft.png":::
+    :::image type="complex" source="./media/pwabuilder-windows-package-options.png" alt-text="Paste publisher information into PWABuilder" lightbox="./media/pwabuilder-windows-package-options.png":::
        Paste publisher information into PWABuilder  
     :::image-end:::  
     
-1.  Choose **Done**.  
+1.  Choose **Generate**.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/microsoft-store.md:110]:
```suggestion
1.  Select **Generate**.  
```

File: microsoft-edge/progressive-web-apps-chromium/microsoft-store.md [hunk 2]
```diff
@@ -95,18 +95,19 @@ Now that you have your app publishing information, generate a Windows app packag
 To generate an app package, complete the following actions.  
 
 1.  Navigate to [PWA Builder][PwabuilderMain].  
-1.  Type the URL of your PWA.  
-1.  Choose **Start** > **Build My PWA** > **Windows** > **Options**.  
+1.  Type the URL of your PWA and click **Start**.  
+1.  Once your report is complete, make sure your PWA is store-ready. If your PWA score is too low, you can visit the **Manifest Options** and **Service Worker Options** and check out the sections that need work.
+1.  When your PWA is ready to be published, click **Next** and select the **Store Package** button in the Windows section of the publish page.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/microsoft-store.md:100]:
```suggestion
1.  When your PWA is ready to be published, click **Next** and select the **Store Package** button in the **Windows** section of the publish page.
```

File: microsoft-edge/progressive-web-apps-chromium/microsoft-store.md [hunk 3]
```diff
@@ -95,18 +95,19 @@ Now that you have your app publishing information, generate a Windows app packag
...
     *   **Publisher Display Name**  
         
-    :::image type="complex" source="./media/pwabuilder-publisher-info.msft.png" alt-text="Paste publisher information into PWABuilder" lightbox="./media/pwabuilder-publisher-info.msft.png":::
+    :::image type="complex" source="./media/pwabuilder-windows-package-options.png" alt-text="Paste publisher information into PWABuilder" lightbox="./media/pwabuilder-windows-package-options.png":::
        Paste publisher information into PWABuilder  
     :::image-end:::  
     
-1.  Choose **Done**.  
+1.  Choose **Generate**.  
 1.  To download your Windows app package, choose **Download**.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/microsoft-store.md:111]:
```suggestion
1.  To download your Windows app package, select **Download**.
```
I don't plan a complete Edit pass of this .md file, b/c this PR is in a fork so I can't do a local edit on the branch and then push my commits.

**Comment by mikehoffms** [comment]
@Maggers - added you as PWA tech reviewer.  I'm not planning on doing a full Edit pass or trying to update the entire article, partly because this PR is in a fork.

**Comment by mikehoffms** [comment]
@MariaSolOsso - I was able to do commits into this PR, even though your working branch is in a fork of the repo.  I'm just wondering if my commits somehow made it into your working branch on the server in your fork.  ("MariaSolOsso" account: master" branch) feeds into this PR.  Maybe my commits (via the PR UI) automatically made it back into your "master" branch in your fork of the repo.

**Comment by MariaSolOsso** [comment]
Hi @mikehoffms, I can indeed see your changes in the master branch of my forked repo

**Comment by vchapel** [comment]
The date is still 4/28/2021, is that accurate? Based on the changes it looks like it should be 8/5 or 8/6 ... or today?

**Comment by mikehoffms** [comment]
> The date is still 4/28/2021, is that accurate? Based on the changes it looks like it should be 8/5 or 8/6 ... or today?

@MariaSolOsso @jgw96 Can we vouch that this overall article is reasonably confirmed to be up-to-date?  If so, please change the date at top to the current date.

File: microsoft-edge/progressive-web-apps-chromium/microsoft-store.md [hunk 4]
```diff
@@ -95,19 +95,20 @@ Now that you have your app publishing information, generate a Windows app packag
 To generate an app package, complete the following actions.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/microsoft-store.md:95]: (For visibility, adding same comment in the Files tab in addition to on the Conversation page.)
Justin/Maria, Can we vouch that this overall article is reasonably confirmed to be up-to-date? If so, please change the date at top to the current date.

**Review by jgw96 — APPROVED** [review]
LGTM! The steps are all correct, use the correct / updated wording and the images are accurate too. Thanks!

**Review by mikehoffms — APPROVED** [review]
Approved per Edit pass.

**Pull request merged**